src/components/Button.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import './Button.css';
+
+class Button extends Component {
+  render() {
+    const { name } = this.props;
+    return (
+      <button className="Button" type="button">
+        {name}
+      </button>
+    );
+  }
+}
```

Comment:
[Optional] (Also may solve the linter isse)

Let's use a function component instead since this componen tis stateless. Something like this like equally work the same way.

```jsx
const Button = ({ name }) => (
      <button className="Button" type="button">
        {name}
      </button>
)
```